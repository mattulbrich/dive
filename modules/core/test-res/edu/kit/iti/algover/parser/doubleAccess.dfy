/* Was a bug in the parser */

class C
{
  var x: C;
  
  method m()
  {
     x.x.x := x.x.x;
  }
}