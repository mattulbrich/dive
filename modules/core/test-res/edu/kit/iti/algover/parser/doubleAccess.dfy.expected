(COMPILATION_UNIT
 (class C
        (FIELD x (TYPE C))
        (method m ARGS
                (BLOCK (:= (FIELD_ACCESS (FIELD_ACCESS x x) x)
                           (FIELD_ACCESS (FIELD_ACCESS x x) x))))))
